### DISCUSSION TITLE
used wrong setting homo sapien and not fly

### DISCUSSION THREAD

**Discussion by Unhingedgene** [Gaius-Augustus/Augustus#281] [Q&A] [answered]
How bad is it if I left Augustus set to homo sapiens but am analyzing an obscure fly that is not close to drosophila. It branches far off in phylo-tree. My boss is pissed but it works when compared to RNA-Seq data.

**Comment by KatharinaHoff** [comment]
This has happened to many of us. Just re-run with the correct species set
and compare results. Best of luck!

**Reply by Unhingedgene** [reply]
What if its the only setting that finds a gene? Drosophila usually produces no results... It's quite strange.

**Comment by KatharinaHoff** [comment, marked as answer]
Try BRAKER2 to retrain for your new fly :-)

**Reply by Unhingedgene** [reply]
thanks
